src/lib/base.jl

```diff
@@ -44,6 +44,27 @@ end
   end
 end
 
+@adjoint function Base._oidd_nextind(a, i)
```

Comment:
do we need to define an adjoint for an internal function of Base?

Reply:
Unfortunately I couldn't see a different way at the time. I'm with you on internal functions. We ended up dropping some grads without it, which hopefully shouldn't be.